src/arno_evap.c

```diff
@@ -85,6 +85,7 @@ double arno_evap(layer_data_struct *layer,
   double max_infil;
   double Evap;
   double tmpsum;
+  double rarc_soil = 100.0;
```

Comment:
@jhamman commented: why 100.0?
@tbohn's reply: 100 gives better results than 0 or 200, from tests I conducted at eddy covariance towers. I didn't do a fancy Bayesian parameter estimation, not enough time for that. I apologize for making this a hard-wired value - I considered making it another input soil parameter, but we have so many of those already. And this likely won't noticeably affect anyone who has been using the traditional VIC set-up, since bare soil rarely occurs in the traditional parameters.